### ISSUE TITLE
Change all __call__ arguments using tf.float32 as a default to the string "float32"

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-cv#279]

**Comment by LukeWood** [comment]
Easy starter issue if anyone wants to familiarize themself with the KerasCV contribution process.

@winstoncooke maybe

**Comment by ricardoprins** [comment]
Is it just changing dtypes from `tf.float32` to `float32`?

**Comment by LukeWood** [comment]
Yeah, it should be the string "float32".  Basically we shouldn't default arguments to an imported symbol.
